test/ibmq/test_ibmq_job_states.py

```diff
@@ -125,6 +125,36 @@ def test_unrecognized_status(self):
         with self.assertRaises(IBMQJobApiError):
             self.wait_for_initialization(job)
 
+    def test_done_status(self):
```

Comment:
I know most of the tests in this class don't have docstrings, but two wrongs don't make a right ;)

Reply:
Yes, I agree completely! 😂